features/comment.feature

```diff
@@ -2,7 +2,8 @@ Feature: Manage WordPress comments through the REST API
 
   Background:
     Given a WP install
-    And I run `wp plugin install rest-api --activate`
+    When I run `wp core version`
+	Then STDOUT should be a version string >= 4.7
```

Comment:
Ok, that's fine. I'll pull code out of your pull request when I have some bandwidth for it.